src/lib/game.svelte

```diff
@@ -68,17 +68,14 @@
 	</div>
 </div>
 
-<div class="absolute left-0 top-0 font-bold">
-		<span class="">
-			Game State: Score: {$gameState.score}
-		</span>
-		<span>
-			Number of moves: {$gameState.numberOfMoves}
-		</span>
-		<span>
-			Combo: {$gameState.comboMultiplier}x
-		</span> 
-	</div>
+<div class="absolute left-0 top-0 font-bold ml-2">
+		<br>
+		Score: {$gameState.score}
+		<br>
+		Number of moves: {$gameState.numberOfMoves}
+		<br>
+		Combo: {$gameState.comboMultiplier}x
+</div>
```

Comment:
This is a WIP, if you want to change this. Please build something that is an actual display of these stats. Don't just change the "debug" stats. 

This is something that could be integrated with the timer if you want to work on that too 
![Untitled](https://github.com/quabey/color-tiles/assets/50772988/722d4c8c-68b9-4f29-a484-646c22f5cb25)


Can you tell that graphic design is my passion ?

Reply:
Alright, I'll keep that in mind, and I'll definently take a look on integrating some better UI when I get a bit more time 😄